Frameworks/Foundation/NSDecimalNumber.mm

```diff
@@ -14,296 +14,360 @@
 //
 //******************************************************************************
 
-#import "Starboard.h"
-#import "StubReturn.h"
-#include <Foundation/Foundation.h>
+#import <Starboard.h>
+#import <StubReturn.h>
+#import <Foundation/Foundation.h>
+#import <Foundation/NSDecimal.h>
+#import <Foundation/NSDecimalNumberHandler.h>
+#import <vector>
+#import <algorithm>
+#import <limits>
 
 NSString* const NSDecimalNumberExactnessException = @"NSDecimalNumberExactnessException";
 NSString* const NSDecimalNumberOverflowException = @"NSDecimalNumberOverflowException";
 NSString* const NSDecimalNumberUnderflowException = @"NSDecimalNumberUnderflowException";
 NSString* const NSDecimalNumberDivideByZeroException = @"NSDecimalNumberDivideByZeroException";
 
+const int _MAX_EXP = 127;
```

Comment:
I'm honestly fine if you put this as a constant in NSDecimal.h (like NSDecimalMaxExponent or something)

---
In reply to: [81210140](https://github.com/Microsoft/WinObjC/pull/1032#discussion_r81210140) [](ancestors = 81210140,81209839)